### PR TITLE
add JSM slides

### PR DESCRIPTION
**Pull request by ywang-gilead**

### CODE REVIEW

**Review by danielinteractive — COMMENTED** [review]
Thanks @ywang-gilead , I wonder if you still have time, could be nice to 1) include results from comparison vignette / simulation study and 2) also mention impact eg stars on GH, CRAN downloads etc

**Comment by ywang-gilead** [comment]
@danielinteractive thanks for the suggestion. I added two more slides at the end.

File: jsm-aug2023.qmd [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+  - S3 method that is compatible with `emmeans`
+  - LS means difference can be produced through `emmeans` (`pairs` method)
+  - Degrees of freedom method is passed from `mmrm` to `emmeans`
+  - By default `PROC MIXED` and `mmrm` do not adjust for multiplicity, whereas `emmeans` does.
+
+## Benchmarking with other R packages
+
+- Compared `mmrm::mmrm` with `nlme::gls`, `lme4::lmer`, `glmmTMB::glmmTMB`
+- Highlights 
+  - `mmrm` has faster convergence time
```
Comment (danielinteractive) [inline jsm-aug2023.qmd:205]: maybe give indication of the difference? i.e. summarizing a bit more details of https://openpharma.github.io/mmrm/main/articles/mmrm_review_methods.html#convergence-times

File: jsm-aug2023.qmd [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+- GitHub repository: 60 stars as of 8/7/2023
+  - [https://github.com/openpharma/mmrm](https://github.com/openpharma/mmrm) 
+
+
+## Outlook
+
+- We still have major features on our backlog
+  - Type II and type III ANOVA tests
+- Working on comparison to other implementations
```
Comment (danielinteractive) [inline jsm-aug2023.qmd:223]: I think this can be removed now since more or less finished

File: jsm-aug2023.qmd [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  - Parameter estimates, computation time, convergence, ...
+- Please let us know what is missing in `mmrm` for you! And try it out :-)
```
Comment (danielinteractive) [inline jsm-aug2023.qmd:225]: you could add another bullet point for new CRAN version 0.3 coming soon and include few additional features from https://github.com/openpharma/mmrm/blob/main/NEWS.md#new-features

**Review by danielinteractive — APPROVED** [review]
thanks Ya, last comments inside then good to merge and present - have fun!!

**Comment by ywang-gilead** [comment]
thanks @danielinteractive just made these updates

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**